src/Notes/WC_Admin_Notes_Personalize_Store.php

```diff
@@ -0,0 +1,70 @@
+<?php
+/**
+ * WooCommerce Admin Personalize Your Store Note Provider.
+ *
+ * Adds a note to the merchant's inbox prompting them to personalize their store.
+ *
+ * @package WooCommerce Admin
+ */
+
+namespace Automattic\WooCommerce\Admin\Notes;
+
+defined( 'ABSPATH' ) || exit;
+
+/**
+ * WC_Admin_Notes_Personalize_Store
+ */
+class WC_Admin_Notes_Personalize_Store {
+	/**
+	 * Note traits.
+	 */
+	use NoteTraits;
+
+	/**
+	 * Name of the note for use in the database.
+	 */
+	const NOTE_NAME = 'wc-admin-personalize-store';
+
+	/**
+	 * Possibly add the note.
+	 */
+	public static function possibly_add_personalize_store_note() {
+		// Only show the note to stores with homepage.
+		$homepage_id = get_option( 'woocommerce_onboarding_homepage_post_id', false );
+		if ( 0 > $homepage_id ) {
```

Comment:
Good call here!  Can we move this logic (zero checking) to the `woocommerce_onboarding_homepage_post_id` creation so we can just check `if ( $homepage_id )` here?

https://github.com/woocommerce/woocommerce-admin/blob/8585ed89b80c394b9cec160ded8c591a8fbfa20d/src/API/OnboardingTasks.php#L416

Would be useful in case we need to check this in other areas and ever get a `0` return value from `wp_insert_post()`.

Reply:
Good idea Josh!
I just added it in the commit [623350f](https://github.com/woocommerce/woocommerce-admin/pull/3895/commits/623350ff91cdc1fe2da1834ae2845333b48bf026)